client/src/Routes.tsx

```diff
@@ -2,7 +2,7 @@ import Home from "./components/Home";
 import SearchMovie from "./components/movieSearch";
 import Twitter from "./components/Twitter";
 import Profile from './components/Profile';
-
+import Discover from "./components/Disover";
```

Comment:
I see that the title of the other file is the same, do you want to keep this, or change it to Discover.tsx?